frontend/my-app/src/index.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+import './index.css';
+
+ReactDOM.render(
+  <App />,
+  document.getElementById('root')
```

Comment:
en [el getting started de reactjs](https://es.reactjs.org/docs/add-react-to-a-website.html) usan este otro código `document.querySelector('#like_button_container')`, no sé si cambia en algo o no

Reply:
Hola! Desconozco, este lo saque del tutorial https://reactjs.org/tutorial/tutorial.html